sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -0,0 +1,123 @@
+import { InjectLogger } from "../../common";
+import {
+  DisbursementSchedule,
+  OfferingIntensity,
+} from "../../database/entities";
+import { LoggerService } from "../../logger/logger.service";
+import {
+  ConfigService,
+  DisbursementScheduleService,
+  SequenceControlService,
+} from "../../services";
+import { getDayOfTheYear, getUTCNow } from "../../utilities";
+import { EntityManager } from "typeorm";
+import { ESDCFileHandler } from "../esdc-file-handler";
+import { ECertFullTimeIntegrationService } from "./e-cert-full-time-integration/e-cert-full-time-integration.service";
+import { ECertFullTimeRequestService } from "./e-cert-full-time-integration/e-cert-full-time-request.service";
+import { ECertUploadResult } from "./e-cert-full-time-integration/models/e-cert-full-time-integration.model";
+import { Injectable } from "@nestjs/common";
+
+const ECERT_SENT_FILE_SEQUENCE_GROUP = "ECERT_SENT_FILE";
```

Comment:
Are the e-Cert files for FT/PT supposed to share the same FILE_SEQUENCE_GROUP?
Since there will be a file for each type I believe that we should have different sequence groups, right?

Reply:
No changed it, they are different.